src/katsdpcontroller/schemas/product_config.json.j2

```diff
@@ -586,6 +586,9 @@
                                 "any_gpu": {"type": "boolean", "default": false},
                                 "disable_ibverbs": {"type": "boolean", "default": false},
                                 "less_resources": {"type": "boolean", "default": false}
+{% if version >= "4.2" %}
+                                "disable_good_data_timeout": {"type": "boolean", "default": false}
```

Comment:
You're missing a comma between the previous item and the next one. You can fix it like this:
```suggestion
                                , "disable_good_data_timeout": {"type": "boolean", "default": false}
```
but I think that looks ugly. I usually try to add new stuff at the top rather than the bottom to avoid this issue.

Something we could consider one day is writing this file in YAML - as long as we stick to the JSON data model and don't use the fancy features of YAML it should make things more readable. It looks like katsdpcontroller sucks in pyyaml indirectly anyway.

Reply:
Makes sense. I'll go with adding at the top for aesthetic reasons.